committeeoversightapp/management/commands/import_data.py

```diff
@@ -18,19 +17,6 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
```

Comment:
Is there any reason why we'd want to import the `house` and `senate` hearings separately (e.g., importing just the house hearings)? If so, it would be worth adding optional arguments: https://docs.djangoproject.com/en/2.1/howto/custom-management-commands/#accepting-optional-arguments

with that said.....I do not *think* there's a reason for this, but thought I'd run it by you, just in case.

Reply:
Not at the moment! There were only 3 hours allocated for this import so I think it makes sense to not implement this until we need to.